ppr-ui/src/components/mhrTransfers/HomeOwnersGroupError.vue

```diff
@@ -23,10 +23,10 @@
   <!-- Transfer Sale or Gift error messages -->
   <span v-else-if="isTransferDueToSaleOrGift && TransSaleOrGift.hasMixedOwnersInGroup(groupId)">
     <span v-if="hasOneHomeOwnerGroup">
-      {{ transfersErrors.hasMixedOwnerTypes }}
+      {{ MhrErrorMsgs.HAS_MIXED_OWNER_TYPES }}
     </span>
     <span v-else>
-      {{ transfersErrors.hasMixedOwnerTypesInGroup }}
+      {{ MhrErrorMsgs.HAS_MIXED_OWNER_TYPES_IN_GROUP }}
     </span>
```

Comment:
Same as above

Reply:
I used ternary here, as the hasOneHomeOwnerGroup is used in multiple spots